src/measurement-events/NinjaFormsEventList.php

```diff
@@ -0,0 +1,30 @@
+<?php
+include_once 'MeasurementEventList.php';
+
+
+/**
+ * Subclass that contains information for Ninja Forms plugin
+ *
+ * @class NinjaFormsEventList
+ */
+class NinjaFormsEventList extends MeasurementEventList {
+
+    public function __construct() {
+        $pluginName = 'Ninja Forms';
+        $categories = array('engagement');
+        $actions = array('form_submit');
+        $selectors = array('div.nf-field-container.submit-container [type="button"]');
+        $onEvents = array('click');
+        $secondLayerSelectors = array('document');
+        $secondLayerOn = array('nfFormReady');
+
+        $tempEvents = array();
+        for($i = 0; $i < count($selectors); $i++) {
```

Comment:
No for loop needed. It's easier to read if the MeasurementEvent creation is inlined

Reply:
Can you explain a little more what you mean by this comment?